src/dmd/escape.d

```diff
@@ -2341,3 +2354,11 @@ private void addMaybe(VarDeclaration va, VarDeclaration v)
         va.maybes = new VarDeclarations();
     va.maybes.push(v);
 }
+
+
+private bool isUnsafe(FuncDeclaration f)
```

Comment:
This function is dip1000-specific, but the name doesn't reflect that. Also, an `isXxx` function should not modify state. How about `setUnsafeDip1000`?